src/microdot.py

```diff
@@ -275,10 +277,7 @@ def json(self):
 
     @property
     def form(self):
-        if self.content_type is None:
-            return None
-        mime_type = self.content_type.split(';')[0]
-        if mime_type != 'application/x-www-form-urlencoded':
+        if self.content_type != 'application/x-www-form-urlencoded':
```

Comment:
This is actually incorrect. The `form()` method returns a parsed form submission. Requests with a JSON body are not form submissions, for such a request this method correctly returns `None`.

The JSON payload can be accessed from the `json()` method of the request object.